### PR TITLE
ignores for code coverage

### PR DESCRIPTION
**Pull request by jpchen**
increasing that code coverage. we're better than 89%

optimizers that are untested are used in examples. presumably theyd be covered if we ran coverage reports in test_examples. (verifying by running the CI, cant do it locally...)

### CODE REVIEW

**Comment by neerajprad** [comment]
> optimizers that are untested are used in examples

It will be nicer to have actual tests than relying on examples for coverage though. Just the examples and tutorials themselves would cover a lot of code but its without any assertions and isn't specific enough to help with fixes.

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ignore:
+  - "pyro/docutil.py"
```
Comment (neerajprad) [inline .codecov.yml]: Nice.

**Comment by jpchen** [comment]
> It will be nicer to have actual tests than relying on examples for coverage though.

yeah i definitely agree with that, i just dont foresee that happening anytime soon. in any case, this is just an experiment to see how codecov works (so i can compare reports). dont need to necessarily merge this

**Comment by neerajprad** [comment]
> yeah i definitely agree with that, i just dont foresee that happening anytime soon.

Good to see the results. I am fine merging this too. I think a big part of the value is in seeing marginal coverage increase/decrease on new PRs, and making sure that we are not merging PRs with large chunks of untested code paths. The overall numbers that we advertise is less of a concern, since its good either ways. 😄

**Comment by neerajprad** [comment]
Apart from `clipped_adam` and `adagrad_rmsprop` in `optim`, the other two files whose coverage improves are example `utils` (not surprising or important since they are meant to be consumed by examples), and `gpr.py` in the GP module.

**Comment by jpchen** [comment]
yeah other than that the largely untested modules are the omt distributions. i thought of `ignore`ing utils but those contain functions that should be tested.  and i agree, i think this at least keeps us honest with respect to testing, though id prefer not having sub-90 in our readme badge.

**Pull request merged**